README.md

```diff
@@ -377,3 +377,8 @@ This CVE has been reported in systems.text.regularexpressions.dll which is used
 * Delete any logs collected thus far and make sure that all copies are deleted. 
 * If you cannot upgrade for any reason, please ensure all debugging is disabled
 * If you are concerned about a potential compromise, contact Snowflake Customer Support for assistance with invalidating all active sessions/tokens. 
+
+3. Global http connection settings - 
+	Snowflake has identified an issue where the connector is globally enforcing TLS 1.2 and certificate revocation checks with connectors up to v1.2.1.
+	Starting with v2.0.0, the connector will set these locally. However note that connector is now targeting .Net framework 4.7.2. 
```

Comment:
Maybe move the "However" to the next paragraph and add a blank line before it. e.g.

```
Starting with ...

Note that v2.0.0 of the driver requires version 4.7.2 (or a more recent version) of the .NET Framework. In addition, when upgrading to v2.0.0, you might need to run "Update-Package -reinstall" to update the dependencies.
```

Reply:
@sfc-gh-dyoshinaga : I think I addressed all the comments, please double-check.

Reply:
Thank you -- these look resolved